mycoffee/__main__.py

```diff
@@ -47,6 +47,8 @@ def main():
     parser.add_argument('--version', help='version', nargs="?", const=1)
     parser.add_argument('--info', help='info', nargs="?", const=1)
     parser.add_argument('--ignore-warnings', help='ignore warnings', nargs="?", const=1)
+    parser.add_argument('--file-path', help='file path to save', type=str)
+    parser.add_argument('--file-format', help='file format', type=str, choices=FILE_FORMATS_LIST, default="text")
```

Comment:
it's not clear it's saving, one'd guess it's for loading.
I suggest `--save-to` and `--save-as`

Reply:
Good point.
Fixed in https://github.com/sepandhaghighi/mycoffee/pull/101/commits/9d03863439399775f3f457a258b0a3f9c11edcf5